iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/MaintainerTreeImplTest.java

```diff
@@ -46,7 +46,9 @@ void matchingSimple() {
       .matches("MAINTAINER bob \\\r\n boberman")
       .matches("MAINTAINER \"bob boberman bob@bob.com\"")
       .matches("MAINTAINER bob /  boberman")
-      .matches("MAINTAINER bob \\ boberman")
+      // TODO : this is a valid syntax that we should support
+      .notMatches("MAINTAINER bob \\ boberman")
+
```

Comment:
This solution has this counterpart : we do not support such instruction anymore, where we have a `\` with spaces before and after, in the middle of a `StringLiteral`.